lib/config/validation-helpers/managers.ts

```diff
@@ -13,7 +13,8 @@ export function check({
   if (Array.isArray(resolvedRule.matchManagers)) {
     if (
       resolvedRule.matchManagers.find(
-        (confManager) => !allManagersList.includes(confManager)
+        (confManager) =>
+          !allManagersList.includes(confManager.replace('custom.', ''))
```

Comment:
```suggestion
          !isKnownManager(confManager)
```
I think we should get a utility function for that.